roles/openshift_master/templates/atomic-openshift-master.j2

```diff
@@ -2,8 +2,9 @@ OPTIONS=--loglevel={{ openshift.master.debug_level }}
 CONFIG_FILE={{ openshift_master_config_file }}
 
 # Proxy configuration
-# Origin uses standard HTTP_PROXY environment variables. Be sure to set
-# NO_PROXY for your master
-#NO_PROXY=master.example.com
-#HTTP_PROXY=http://USER:PASSWORD@IPADDR:PORT
-#HTTPS_PROXY=https://USER:PASSWORD@IPADDR:PORT
+# See https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html#configuring-global-proxy
+{% if openshift.common.http_proxy or openshift.common.https_proxy %}
+HTTP_PROXY='{{ openshift.common.http_proxy }}'
+HTTPS_PROXY='{{ openshift.common.https_proxy }}'
+NO_PROXY='{{ openshift.common.no_proxy }}'
+{% endif %}
```

Comment:
Should we keep the commented examples in the file in the case that we aren't setting the proxy values?